src/utils/url.js

```diff
@@ -31,7 +31,18 @@ function _convert (v) {
   return _isBoolean(_isNull(_isNum(v)))
 }
 
-export function urlParams (url) {
+function _parseParam (params, key) {
+  if (params[key]) {
+    if (isArray(params[key])) {
+      params[key] = params[key].map(v => _convert(_decode(`${v}`)))
+    } else {
+      params[key] = _convert(_decode(`${params[key]}`))
```

Comment:
You now should be able to get rid of the string building part here.

Reply:
done